website/widgets/form/table.md

```diff
@@ -355,17 +355,20 @@ Check the [complete example](/docs/examples/table-pagination) with all the suppo
 
 ## With links
 
-You can have links using Markdown notation and setting `data_type` to `'link'`.
+Links can be displayed by setting `cell_type` to `'ui.url_table_cell_type'`. If `'new_tab'` is set to `'True'` the link will be opened on a new tab.
+To provide a label use the Markdown notation `[label](url)`.
+If the URL doesn't have '`http(s)://`' the link will be appended to the current URL, trying to access a page in the same domain.
```

Comment:
@mturoci 
> Can you provide a link? I was not able to find the string you quoted in table.md

Current behavior
1. example.com -> <current_domain>/example.com
2. http(s)://example.com -> example.com

Maybe we should remove the "trying to access a *page* in the same domain" part because it might confuse people with wave's concept of *page*.

Reply:
The behavior is fine (that's how the web works). Changing it would be a breaking change which I am not sure is worth it. However, you are right that the wording can be improved to not interfere with the page concept in Wave - +1.